src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -29,7 +29,7 @@
         "symfony/config": "^6.4|^7.0",
         "symfony/console": "^6.4|^7.0",
         "symfony/css-selector": "^6.4|^7.0",
-        "symfony/dependency-injection": "^6.4|^7.0",
+        "symfony/dependency-injection": "^7.1",
```

Comment:
is there a way to prevent this? I'd like to preserve compat with 6.4 for as long as possible

Reply:
I pushed an alternate solution. We can actually simply remove this bit that depends on the new `prepend` argument as well as the test assertion that goes with it. I think it is safe to do so because the `prepend` argument is tested here:

https://github.com/symfony/symfony/blob/638f7d0840e838ef5183eeab622892f56c3cd8b2/src/Symfony/Component/DependencyInjection/Tests/Extension/AbstractExtensionTest.php#L57-L79

What do you think?